### DISCUSSION TITLE
add Folders to "Data"

### DISCUSSION THREAD

**Discussion by jpeter06** [mockoon/mockoon#1037] [Ideas]
Hello All,
folders are wonderfull for routes. Very nice way of grouping mocks.
It is posible to add the same functionality to de "Data" list

thanks for all. CRUD save muy time :-)

**Comment by 255kb** [comment]
Glad you like the CRUD feature! it's an industry standard that was definitely missing. 

It's true folders for data buckets could be great. It's not planned though. Could you please open an issue? Thank you.
